runtime/vm/jfr.cpp

```diff
@@ -1174,7 +1174,7 @@ jfrDump(J9VMThread *currentThread, BOOLEAN finalWrite)
 {
```

Comment:
the RI doesnt output this event all the time, so I think we should only output it when dump is explicitly called. Coincidentally, this is also when we have exclusive VMaccess. 

So instead of `writeOutGlobalBuffer(J9VMThread *currentThread, bool finalWrite, bool isExclusivePermited)` maybe just `writeOutGlobalBuffer(J9VMThread *currentThread, bool finalWrite, bool dumpCalled)`

Reply:
But the [JFR Event Collection website](https://sap.github.io/SapMachine/jfrevents/21.html#javathreadstatistics) says it's "every chunk" and my local testing recording file created by RI with `-XX:+FlightRecorder -XX:StartFlightRecording=filename=flight.jfr` also contains this event.

Reply:
We generate more chunks then RI since we do it everytime a thread dies and on every GC. So its not a apples-to-apples comparison. It makes sense to do the long running "everychunk" operations only when dump is called.